src/pages/patientView/PatientViewPage.tsx

```diff
@@ -38,10 +38,16 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
             );
         }
 
+        let samples = ["string", "TCGA-24-2024-01", "TCGA-24-2024-01 (deliberately elongated id)"];
         let mutationDiv: Element | null = document.getElementById('mutations_div_prototype');
         if (mutationDiv) {
             ReactDOM.render(
-                <MutationInformationContainer {...{store: this.props.store}} />,
+                <MutationInformationContainer sampleOrder={samples}
+                                             sampleLabels={samples.reduce((map, sample, i) => {map[sample] = i+""; return map;}, {})}
+                                            sampleColors={samples.reduce((map, sample) => {map[sample] = (Math.random() > 0.5 ? "black": "red"); return map;}, {})}
+                                              sampleTumorType={samples.reduce((map, sample) => {map[sample] = (Math.random() > 0.5 ? "Primary": "Metastasis"); return map;},{})}
+                                              sampleCancerType={samples.reduce((map, sample) => {map[sample] = "Cancer Type of " + sample; return map;}, {})}
+                                            {...{store: this.props.store}} />,
```

Comment:
I guess this is more of a design issue. We still don't know how/where to get additional data (other than the core mutation data). Passing this information all the way down to the children as props might be one solution, or fetching this information within the designated column renderer might be another solution. By fetching I don't necessarily mean fetching it directly from the server side. That information might have already been fetched by another component (in this case, `PatientViewPage`) and stored in some sort of global/shared cache/store or something. 

At this point, I don't really know what the best design is for this kind of additional/shared data. Maybe we should discuss this with @alisman for a better decision on this.

But for now, we need to mock the data somewhere else, not hard-coded or generated within this component.